src/components/Settings/SettingsServices.tsx

```diff
@@ -283,10 +283,11 @@ const SettingsServices: React.FC = () => {
                   }
                 />
               ))}
-              <li className="h-32 col-span-1 border-2 border-gray-400 border-dashed rounded-lg shadow sm:h-32">
+              <li className="h-46 col-span-1 border-2 border-gray-400 border-dashed rounded-lg shadow">
                 <div className="flex items-center justify-center w-full h-full">
                   <Button
                     buttonType="ghost"
+                    className="mt-3 mb-3"
                     onClick={() =>
                       setEditRadarrModal({ open: true, radarr: null })
                     }
```

Comment:
Why did we change this?

Before:
![image](https://user-images.githubusercontent.com/234213/106378095-2be4af00-63e5-11eb-81dd-16c2c4f7a1f1.png)


Now:
![image](https://user-images.githubusercontent.com/234213/106378088-11123a80-63e5-11eb-8d7b-ffddbebbf5b7.png)